setup.py

```diff
@@ -35,24 +37,54 @@
 # stdlib, which is deprecated in Python 3.10 and will be removed in 3.12.
 from distutils import log  # isort: skip
 
+
 # Controls whether providers are installed from packages or directly from sources
 # It is turned on by default in case of development environments such as Breeze
 # And it is particularly useful when you add a new provider and there is no
 # PyPI version to install the provider package from
 INSTALL_PROVIDERS_FROM_SOURCES = 'INSTALL_PROVIDERS_FROM_SOURCES'
-PY39 = sys.version_info >= (3, 9)
+PY39 = sys.version_info >= (
+    3,
+    9,
+)
```

Comment:
This looks weird with an element per line :)

```suggestion
PY39 = sys.version_info >= (3, 9)
```

Reply:
I think black formatted it - but I can check :). I did not do it it for sure on purpose :)

Reply:
Something else did :) - but not sure what :)